django/thunderstore/frontend/models.py

```diff
@@ -13,6 +13,8 @@ class DynamicPlacement(ChoiceEnum):
     html_body_beginning = "html_body_beginning"
     content_beginning = "content_beginning"
     content_end = "content_end"
+    left_main_column = "left_main_column"
+    right_main_column = "right_main_column"
```

Comment:
I'd prefer the term ordering here would be the other way around, in the CSS also.
```
main_colum_left
main_column_right
```

This is a good general convention and not just limited to this specific case. Some reasons for why:
- It keeps the variable names associated with the same concept together e.g. when sorting by alphabet
- If using autofill with IDEs (you can just start writing `main_colum_` to have the IDE suggest autofills rather than having to remember what specific keyword to prefix first).
- It's easier to visually identify them as relating to each other if they share the same prefix

You can see the same convention applied elsewhere in the codebase (though not universally), including in this enum itself (`html_head`, `html_body_beginning`, `content_beginning`, `content_end`, etc.)

Reply:
Good that you pointed this out 👍 I just kinda put something there, since I didn't want to think about naming stuff too much, but you have a good logical way of thinking about the needs of naming stuff correctly. I've fixed the namings and migrations accordinly.